### DISCUSSION TITLE
Meet face to face at ICCC?

### DISCUSSION THREAD

**Discussion by jgb1128** [CC-in-the-Cloud/Admin#94] [General]
Team,
Today there was an announcement on the CCUF Only office that the dates for the CCUF are Oct 26-27 and Oct 30th. I would like to propose that we ask for a closed session space Oct 26-27 at the workshop. Thoughts?

**Comment by GY5318** [comment]
I can make it.  At least it is close, and I don't have to ask for TDY.

Regards

Greg

 

 

From: Josh Brickman ***@***.***> 
Sent: Wednesday, May 17, 2023 1:36 PM
To: CC-in-the-Cloud/General ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [CC-in-the-Cloud] Meet face to face at ICCC? (Discussion #94)

 

Team,
Today there was an announcement on the CCUF Only office that the dates for
the CCUF are Oct 26-27 and Oct 30th. I would like to propose that we ask for
a closed session space Oct 26-27 at the workshop. Thoughts?

-
Reply to this email directly, view it on GitHub
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Forgs%2FCC-in-the-Cloud%2Fdiscussions%2F94&data=05%7C01%7Cgregory.f.yous
t.civ%40mail.mil%7C7a396b38f5bd4c889b6308db56fd2b12%7C102d0191eeae4761b1cb1a
83e86ef445%7C0%7C0%7C638199417584667017%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4w
LjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata
=Ag4%2BKk8fDzf1ZbKGI%2FqT09f2uarebnQmWYlTD4nhMlA%3D&reserved=0> , or
unsubscribe
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Fnotifications%2Funsubscribe-auth%2FAU6AIDWSWDO3VTRDZDJOU7DXGUD7JANCNFSM
6AAAAAAYFNHTEI&data=05%7C01%7Cgregory.f.youst.civ%40mail.mil%7C7a396b38f5bd4
c889b6308db56fd2b12%7C102d0191eeae4761b1cb1a83e86ef445%7C0%7C0%7C63819941758
4667017%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6
Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OaopU2CGK0Yl4zlMrUD8R8gFRS00Sq7
hPbWQqXZUZHE%3D&reserved=0> .
You are receiving this because you are subscribed to this thread.
<https://github.com/notifications/beacon/AU6AIDWH5OCLS4BGSQSSP2DXGUD7JA5CNFS
M6AAAAAAYFNHTEKWGG33NNVSW45C7OR4XAZNKIRUXGY3VONZWS33OVJRW63LNMVXHIX3JMTHAAT3
RTY.gif> Message ID: ***@***.***
***@***.***> >

**Comment by wo-peter** [comment]
Count me in.
- Wolfgang

Wolfgang Peter
Principal Security Program Manager
Azure Databases Security Fundamentals & Compliance

phone: +49 221 80102134
cell: +49 151 44063333
mail: ***@***.***

Microsoft Deutschland GmbH | Walter-Gropius-Straße 5 | 80807 München | www.microsoft.com/germany | http://news.microsoft.com/de-de/
Geschäftsführer/General Managers: Marianne Janik (chair), Florian Deter, Benjamin O. Orndorff, Keith Dolliver | Amtsgericht München, HRB 70438
________________________________
Von: Josh Brickman ***@***.***>
Gesendet: Wednesday, May 17, 2023 7:35:48 PM
An: CC-in-the-Cloud/General ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [CC-in-the-Cloud] Meet face to face at ICCC? (Discussion #94)


Team,
Today there was an announcement on the CCUF Only office that the dates for the CCUF are Oct 26-27 and Oct 30th. I would like to propose that we ask for a closed session space Oct 26-27 at the workshop. Thoughts?

**Reply by wo-peter** [reply]
@jgb1128 
Unfortunately I need to cancel my attendance, as I was on a sick-leave for quite a while. However, I am feeling way better now and plan to attend the ICCC, but not the CCUF. My apologies. 
- Wolfgang

**Comment by GallagherTom** [comment]
I'd like to participate in the closed session as well. Thanks Josh!

**Comment by fahmed11** [comment]
I am planning to attend.

**Comment by jgb1128** [comment]
**Running List of Planned Attendees:**

1. Joshua Brickman-Oracle
2. Brandon Harvey-Oracle
3. Greg Youst-DISA
4. Farid Ahmed-JHU
5. Tom Gallagher-DoD
6. Tyrone Stodart-Oracle
7 Josh Anzaroot-Excentium
8. Jerry Myers-Aerospace
9. Randy Heimann-Mitre
10. Justin Fisher-Leidos
11. Alicia Squires-AWS
12. Cory Clark-Canadian Centre for Cybersecurity
13. Garrett Nickel-Lightship
14. Dylan Kehs-NIAP
15. Thibaut Marconnet-Cisco
16. Dani Chappell--NIAP
17. Vann Nguyen-VMWare
18. Benjamin Jacoby-VMWare
19. Anne Gugel-JHU
20. Mike Grimm-MSFT
21. Shawn Gilmore-MSFT
22. Swapna Katikaneni-NIAP
23. Jade Stewart-NIAP

**Assumed but not confirmed:**

Andy Nissan



**Participate Virtually:**

1. Tim Chevalier
2. Fernie Fuentes
3. Jenn Honkofsky
4. Michael Angelo

**Reply by mfa00007** [reply]
As of right now, I will be there..

**Reply by heimannrj** [reply]
I can likely do at least one day.  Actualy helpful for me to be offsite because I am not going to CCUF.

**Comment by jgb1128** [comment]
Get Metro info and post. Also what about parking.

**Reply by jgb1128** [reply]
The address is:
901 F Street N.W.
Suite 800
The Gallup Building
Washington D.C, District of Columbia  20004

**Comment by GY5318** [comment]
Thanks.  Will see what works with Metro.  Does your building have free
parking if I drive, or do I find a parking garage?

Regards

Greg

 

 

From: Josh Brickman ***@***.***> 
Sent: Monday, July 31, 2023 3:36 PM
To: CC-in-the-Cloud/General ***@***.***>
Cc: Youst, Gregory F CIV DISA SD (USA) ***@***.***>;
Comment ***@***.***>
Subject: Re: [CC-in-the-Cloud/General] Meet face to face at ICCC?
(Discussion #94)

 

The address is:
901 F Street N.W.
Suite 800
The Gallup Building
Washington D.C, District of Columbia 20004

-
Reply to this email directly, view it on GitHub
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Forgs%2FCC-in-the-Cloud%2Fdiscussions%2F94%23discussioncomment-6598641&d
ata=05%7C01%7Cgregory.f.youst.civ%40mail.mil%7C851b0ac937714612455208db91fd5
ff9%7C102d0191eeae4761b1cb1a83e86ef445%7C0%7C0%7C638264289676771364%7CUnknow
n%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6
Mn0%3D%7C3000%7C%7C%7C&sdata=ahO3mHFOkvUGAtT8HKdH9OJbeaiUAaVvDJkaPLidVeo%3D&
reserved=0> , or unsubscribe
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Fnotifications%2Funsubscribe-auth%2FAU6AIDTQC35NXVT7CF2MTGDXTACJTANCNFSM
6AAAAAAYFNHTEI&data=05%7C01%7Cgregory.f.youst.civ%40mail.mil%7C851b0ac937714
612455208db91fd5ff9%7C102d0191eeae4761b1cb1a83e86ef445%7C0%7C0%7C63826428967
6771364%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6
Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=WwL%2BTd4NBD4gB2np362xlpEHOCZNf
zJDXSS3zJHp33k%3D&reserved=0> .
You are receiving this because you commented.
<https://github.com/notifications/beacon/AU6AIDT2USFGNTO6XLZ2FZTXTACJTA5CNFS
M6AAAAAAYFNHTEKWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAMSX7C.gif> Message ID:
***@***.***
***@***.***
com> >

**Reply by bharveyTX** [reply]
There is a parking garage at The Gallup Meeting but Oracle does not have any spots reserved for employees or guests. It looks like the price will be around [$17 a day](https://spothero.com/search?kind=destination&id=39737&starts=2023-10-26T08%3A00&ends=2023-10-26T18%3A00&midfunnel=true&spot-id=40839). The office is about three blocks east of the Metro Center station.

**Comment by jhonkofsky** [comment]
I would like to attend.

**Reply by jgb1128** [reply]
Jenn, per your email, I have removed you from the roster

**Comment by gnickells** [comment]
Confirmed, I will be attending.

**Comment by tmarconn** [comment]
I will be attending as well. 

Thibaut Marconnet
